examples/logger.py

```diff
@@ -3,20 +3,57 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+
+class Signaller(QObject):
+    """Signal for LoggingHandler.
+
+    Signals:
+        signal(log): A formatted log message is emitted. 
+    """
+
+    signal = pyqtSignal(str)
+
+
+class LoggingHandler(logging.Handler):
+    """Handler for logger.
+
+    Sends a log message to connected function using emit.   
+    """
+
+    def __init__(self, slotfunc):
+        """Extended.
+
+        Connects the slotfunc to the signal.
+
+        Args:
+            slotfunc: method function connected to signal.
+        """
+        super().__init__()
+        self.signaller = Signaller()
+        self.signaller.signal.connect(slotfunc)
+
+    def emit(self, record: logging.LogRecord):
+        """Emits input signal to connected function."""
```

Comment:
Since this _overrides_ the method `logging.Handler.emit()`, you should start it with `"""Overridden."""`.
For example:
```python
"""Overridden.

Emits ...
"""
```